chalicelib/utils.py

```diff
@@ -166,6 +155,20 @@ def _make_payload(self):
 
 # Standalone utility functions.
 
+def requests_response_to_chalice_Response(response):
+    """Takes some attributes from a request response object and creates
+    a Chalice Response object from it.
+    
+    :param response: (response object) from requests module 
+    :param status_code: (int) default is 200
```

Comment:
There is no status_code parameter

Reply:
yes, good catch, I'll correct it before merging

Reply:
Hi Charles, I've made a few more small changes. Please take a look.